auth_oauth_force_oauth/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 Quartile Limited (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    force_oauth_domains = fields.Text(help="Fill in the domains, separated by commas.")
```

Comment:
Forgot to ask earlier. Why should this field be `Text` if domains should be separated by commas?

Reply:
You right. It should be `Char` field.